Resources/Private/JavaScript/dev-server/src/server.ts

```diff
@@ -8,6 +8,7 @@ import express from 'express';
 import { createProxyMiddleware } from 'http-proxy-middleware';
 
 import * as Fixtures from './fixtures/index';
+import { FileUploadResult } from '@media-ui/feature-asset-upload/typings';
```

Comment:
Please remove any import from the typings in this PR. You don't need them if you also don't prefix the type with `export`.

Reply:
I adjusted the imports and exports.